app/locale/nl-NL.coffee

```diff
@@ -551,11 +551,11 @@ module.exports = nativeDescription: "Nederlands (Nederland)", englishDescription
 
   subscribe:
     comparison_blurb: "Verbeter je vaardigheden met een abonnement op CodeCombat!"
-    feature1: "meer dan $t(data.levelsCount) basislevels over $t(data.worldsCount) werelden" # {change}
+    feature1: "+$t(data.levelsCount) basislevels over $t(data.worldsCount) werelden" # {change}
     feature2: "$t(data.heroesCount) sterke <strong>nieuwe helden</strong> met unieke vaardigheden!" # {change}
     feature3: "$t(data.bonusLevelsCount) bonuslevels" # {change}
```

Comment:
Can you just update these strings to match the English version? Otherwise, the numbers won't show up.